Library/test-helpers/lib.sh

```diff
@@ -1134,10 +1134,18 @@ true <<'=cut'
 Run 'keylime_tenant -c status' wrapper repeatedly up to TIMEOUT seconds
 until the expected agent status is returned.
 
-    limeWaitForAgentStatus UUID STATUS [TIMEOUT]
+    limeWaitForAgentStatus [--container] [NAME] UUID STATUS [TIMEOUT]
```

Comment:
What I do not like about the `--container` option is that it could be misleading. One could also read it that the agent is running in a container. I would propose to use something like `--tenant-container`, even though it is longer. We could eventually support `-t` or `--tc` as a shortcut. Also, instead of variable `NAME` in the code I would also prefer something like `TENANT_CONTAINER` or `TENANT_CONT_NAME`. Just something to consider.